tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py

```diff
@@ -11,50 +11,146 @@
     preparsing_validators=[
         validators.notEmpty(start=19, end=60),
     ],
-    postparsing_validators=[],
+    postparsing_validators=[
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                  result_field='SSN', result_function=validators.notOneOf(("000000000", "111111111", "222222222",
```

Comment:
Nitpick: I see this SSN validation is used in all these t1-t3 models -- not sure if we'll be using it in subsequent sections like ssn/etc but can we optimize this notOneOf function to take a list and use this? Or make a specific SSN validator that wraps `notOneOf` and provides these numbers?

Some code-golf also:
`[str(i)*9 for i in range(0,9)]`

Reply:
Not sure if I understand but technically, "notOneOf" takes both list and tuples. I am a bit hesitant to create specific validator for SSN, but can use your code snippet.